packages/web-components/fast-foundation/src/design-token/design-token.spec.ts

```diff
@@ -640,7 +715,7 @@ describe("A DesignToken", () => {
             await DOM.nextUpdate();
             expect(handleChange).to.have.been.called.once;
         });
-        it("should notify a subscriber for a token after being appended to a parent with a different token value than the previous context", async () => {
+        it("should notify a subscriber for a derived token after being appended to a parent with a different token value than the previous context", async () => {
```

Comment:
Just disambiguating this test from another right next to it that had the same name/descriptor.